<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerDialog</class>
 <widget class="QDialog" name="VerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit location</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ScriptEdit" name="actionsEdit">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="tabStopWidth">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="verIdEdit"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QRadioButton" name="winRadio">
           <property name="text">
            <string>Win</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="odinaryRadio">
           <property name="text">
            <string>Odinary</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="startRadio">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="failRadio">
           <property name="text">
            <string>Fail</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScriptEdit</class>
   <extends>QTextEdit</extends>
   <header>ScriptEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
